packages/odyssey-react-mui/src/Button.tsx

```diff
@@ -142,6 +150,24 @@ export type ButtonProps = {
     | "translate"
   >;
 
+// Styled version of the Button for gray background context style overrides
+const nonForwardedProps = ["isGrayBackground", "odysseyDesignTokens"];
+
+const StyledButton = styled(MuiButton, {
+  shouldForwardProp: (prop) => !nonForwardedProps.includes(prop as string),
+})<{
```

Comment:
What's this function and why are we doing `as string`? I mean, I understand why relatively, but I wonder if there's a better way of doing this.

Reply:
I agree, that's a bit wonky. I updated it to be more type-safe approach:
  `shouldForwardProp: (prop: string) => !nonForwardedProps.includes(prop),`